admin/src/router/router.test.js

```diff
@@ -44,8 +44,8 @@ describe('router', () => {
     })
 
     describe('routes', () => {
-      it('has seven routes defined', () => {
-        expect(routes).toHaveLength(8)
+      it('has eight routes defined', () => {
```

Comment:
nine?

Reply:
Right.
This has been corrected in https://github.com/gradido/gradido/commit/7c4e2527574fe1528fd4703f37f032b4a2d401e1